cumulus/parachains/integration-tests/emulated/tests/people/people-rococo/src/tests/reap_identity.rs

```diff
@@ -0,0 +1,337 @@
+			RococoOrigin::root(),
+			RococoRelaySender::get()
+		));
+
+		let remote_deposit = calculate_remote_deposit(id.relay.encoded_size() as u32, 1);
+		assert_expected_events!(
+			RococoRelay,
+			vec![
+				RuntimeEvent::Balances(BalancesEvent::Unreserved { who, amount }) => {
+					who: *who == RococoRelaySender::get(),
+					amount: *amount == total_deposit,
+				},
+			]
+		);
+		assert!(PeopleRococoIdentity::identity(&RococoRelaySender::get()).is_none());
+		let (_, sub_accounts) = RococoIdentity::subs_of(&RococoRelaySender::get());
+		assert_eq!(sub_accounts.len(), 0);
+
+		let reserved_balance = RococoBalances::reserved_balance(RococoRelaySender::get());
+		// after reap reserved balance should be 0
+		assert_eq!(reserved_balance, 0);
+		let free_bal_after_reap = RococoBalances::free_balance(RococoRelaySender::get());
+
+		// free balance after reap should be greater than before reap
+		assert!(free_bal_after_reap > free_bal_before_reap);
+		assert_eq!(free_bal_after_reap, free_bal_before_reap + total_deposit - remote_deposit);
+	});
+}
+fn assert_reap_parachain(id: &Identity) {
+	// 6. assert on Parachain
+	PeopleRococo::execute_with(|| {
+		type RuntimeEvent = <PeopleRococo as Chain>::RuntimeEvent;
+		let reserved_bal = PeopleRococoBalances::reserved_balance(PeopleRococoSender::get());
+		let id_deposit = id_deposit_parachain(&id.para);
+		let subs_deposit = SubAccountDepositParachain::get();
+		let total_deposit = subs_deposit + id_deposit;
+
+		assert_expected_events!(
+			PeopleRococo,
+			vec![
+				RuntimeEvent::Balances(BalancesEvent::Deposit { .. }) => {},
+				RuntimeEvent::Balances(BalancesEvent::Endowed { .. }) => {},
+				RuntimeEvent::Balances(BalancesEvent::Reserved { who, amount }) => {
+					who: *who == PeopleRococoSender::get(),
+					amount: *amount == id_deposit,
+				},
+				RuntimeEvent::Balances(BalancesEvent::Reserved { who, amount }) => {
+					who: *who == PeopleRococoSender::get(),
+					amount: *amount == subs_deposit,
+				},
+				RuntimeEvent::IdentityMigrator(
+					polkadot_runtime_common::identity_migrator::Event::DepositUpdated {
+						who, identity, subs
+					}) => {
+					who: *who == PeopleRococoSender::get(),
+					identity: *identity == id_deposit,
+					subs: *subs == subs_deposit,
+				},
+				RuntimeEvent::MessageQueue(pallet_message_queue::Event::Processed { ..}) => {},
+			]
+		);
+
+		// reserved balance should be equal to total deposit calculated on the Parachain
+		assert_eq!(reserved_bal, total_deposit);
+
+		// Should have at least one ED after in free balance after the reap.
+		assert!(PeopleRococoBalances::free_balance(PeopleRococoSender::get()) >= PEOPLE_ROCOCO_ED);
+	});
+}
+
+fn calculate_remote_deposit(bytes: u32, subs: u32) -> Balance {
+	let para_basic_deposit = deposit(1, 17) / 100;
+	let para_byte_deposit = deposit(0, 1) / 100;
+	let para_sub_account_deposit = deposit(1, 53) / 100;
+	let para_existential_deposit = EXISTENTIAL_DEPOSIT / 10;
```

Comment:
I see you (correctly) imported `rococo_runtime_constants::currency::*;`. Still, I think it's worth a comment here since these `deposit` functions are all named the same. Make it clear here that this is the Relay Chain's `currency` because that's the context this function is executed in.